test/etsdb_test.dart

```diff
@@ -245,28 +231,17 @@ void main() {
 
         var firstTime = DateTime.parse(history[1].updates.first[0]);
```

Comment:
Just rename firstTime rather than assign a variable that's just used to assign to another variable (previousTime) and isn't used again.